src/managers/Driver.java

```diff
@@ -360,7 +360,9 @@ public void findBookTitle(String title) {
 		//find error case of title not matching
 		boolean match = false;
 		for (int i = 0; i < bookList.size(); i++) {
-			if (bookList.get(i).getTitle().toLowerCase().contains(title)||bookList.get(i).getTitle().toLowerCase().equals(title)) {
+			//The line below contains the issue to fix
+			//Find text with Contains OR find text with the full title
+			if (bookList.get(i).getTitle().toLowerCase().contains(title)||bookList.get(i).getTitle().toLowerCase().equals(title)) { 
```

Comment:
add .toLowerCase() to tittle on the right side or ||

Reply:
Does that fix the error?

Reply:
that will make it easier for the client to enter a book name with out being case sensitive.